solr/core/src/test-files/solr/collection1/conf/schema12.xml

```diff
@@ -238,6 +227,18 @@
     </analyzer>
   </fieldType>
 
+  <fieldType name="text_pick_best_boost_by_payload" class="solr.TextField" positionIncrementGap="100" synonymQueryStyle="pick_best" autoGeneratePhraseQueries="true">
```

Comment:
You can remove "payload" everywhere from this PR now; no?

Reply:
Fixed in the next coming commit!